### ISSUE TITLE
Android 7.0 Uav Camera Problem

### ISSUE THREAD

**Issue by Zainpeng** [DroidPlanner/Tower#1853]
In Android 7.0，Display function can not be used for uav camera. Need to consider updating.

**Comment by Linjieqiang** [comment]
Yes.The problem is the uvc version was too old and need to update.
